frontend/src/store/modules/compare.js

```diff
@@ -35,9 +35,6 @@ const actions = {
   resetComparisons({ commit }) {
     commit('setComparisons', []);
   },
-  resetComparisonDetails({ commit }) {
```

Comment:
When playing around on the Comparison page, I managed to end up in confusing situations twise.

First, the table contains outdated information.
<img width="441" alt="image" src="https://user-images.githubusercontent.com/23480589/224999885-e9cef554-31e2-47ad-b505-42ed58784bc4.png">

Second, the comparison details on the right do not match what's on the left side.
<img width="852" alt="image" src="https://user-images.githubusercontent.com/23480589/224999579-b56d46b3-3207-433a-a4b1-6d8d5a82504b.png">

If this has not been introduced by these changes, please report as a new issue.